requirements-dev.txt

```diff
@@ -2,6 +2,7 @@ wheel==0.38.1
 behave==1.2.5
 jsonschema==2.5.1
 coverage==5.5
+setuptools==67.8.0;python_version>="3.12"
```

Comment:
What happens if we don't pin `setuptools` to this version for 3.12?

Reply:
`setuptools` is no longer provided with Python starting in Python 3.12. We need to explicitly add it as a dependency.